.github/workflows/release.yml

```diff
@@ -40,13 +40,13 @@ jobs:
             #   run: echo 'TAG_NAME=prerelease' >> $GITHUB_ENV
             - if: github.event_name == 'workflow_dispatch'
               run: echo "TAG_NAME=${{ github.event.inputs.tag_name }}" >> $GITHUB_ENV
-            - if: github.ref_name != 'master'
+            - if: github.ref_name != 'master' || github.ref_name != 'main'
               run: |
                   TAG_NAME=${{ github.ref_name }}
                   FEAT_NAME=$(echo $TAG_NAME | sed 's/feature\///')
                   echo "FEAT_NAME=$FEAT_NAME" >> $GITHUB_ENV
                   echo "TAG_NAME=pre-$FEAT_NAME" >> $GITHUB_ENV
-            - if: github.ref_name == 'master'
+            - if: github.ref_name == 'master' || github.ref_name == 'main'
```

Comment:
these should be `&&` ?

Reply:
Thanks for the corrections.
Yes, for the `!==` we should have the `&&` condition

Reply:
as per

```
- if: github.ref_name == 'master' || github.ref_name == 'main'
   run: |
       echo "FEAT_NAME=" >> $GITHUB_ENV
       echo "TAG_NAME=prerelease" >> $GITHUB_ENV
```

should be fine since it's either

Reply:
i'm more concerned about the semantics, rather than the fact that it happens to work ok